packages/modules/test-local-extension/package.json

```diff
@@ -13,14 +13,21 @@
     "@itwin/core-extension": "^3.2.0"
   },
   "devDependencies": {
-    "typescript": "~4.4.0",
     "@esbuild-plugins/node-modules-polyfill": "~0.1.2",
     "@esbuild-plugins/node-globals-polyfill": "~0.1.1",
+    "@itwin/appui-abstract": "^3.2.0",
     "@itwin/build-tools": "^3.2.0",
+    "@itwin/core-bentley": "^3.2.0",
+    "@itwin/core-common": "^3.2.0",
+    "@itwin/core-geometry": "^3.2.0",
+    "@itwin/core-orbitgt": "^3.2.0",
+    "@itwin/core-quantity": "^3.2.0",
+    "@itwin/webgl-compatibility": "^3.2.0",
     "esbuild": "^0.13.13",
+    "npm-run-all": "^4.1.5",
     "rimraf": "^3.0.2",
     "serve": "^13.0.2",
-    "npm-run-all": "^4.1.5"
+    "typescript": "~4.4.0"
```

Comment:
its actually using the same ver as before, i just alphabetized the deps in addition to adding the missing peers.

using ~ instead of ^ to make sure the entire monorep (our and cores) all use the same ver of ts
we are a few versions behind so we can update ts across this monorepo before next or after, up to you

Reply:
ok yeah I guess to keep with core I guess. Really we should never be using ~. Didnt even realize we were doing that

Reply:
ts is weird, where every minor release its their beta release, and their ga/official release is of the .2 patch for that minor

i guess it wouldnt hurt to use ^ as most things id worry about in a ts upgrade would probably need to be opted in in the tsconfig file, but not 100% sure. those guys do have breaking changes in minor releases too